src/components/tooltip/_tooltip.scss

```diff
@@ -1,31 +1,122 @@
-@import '~styles/variables';
-
-.tooltip {
+.player .tooltip {
+  position: relative;
   display: inline-block;
-  height: 22px;
-  border-radius: 4px;
-  background-color: $tooltip-bg-color;
-  padding: 3px 13px;
-  color: $tooltip-color;
-  font-size: 13px;
-  font-weight: bold;
-  line-height: 16px;
-  box-shadow: 0 0 8px 0 rgba(0, 0, 0, 0.3);
-  position: absolute;
-  transform: translate3d(-50%, 0, 0);
-  left: 50%;
-  bottom: 15px;
-
-  &:after {
-    content: ' ';
-    display: block;
+  .tooltip-label {
+    width: max-content;
+    visibility: hidden;
+    background-color: $tooltip-bg-color;
+    color: $tooltip-color;
+    text-align: center;
+    padding: 4px 6px;
+    border-radius: 4px;
     position: absolute;
-    width: 10px;
-    height: 10px;
-    background-color: #fff;
-    border-radius: 3px;
-    transform: rotate(45deg) translate3d(0%, 50%, 0);
-    left: 50%;
-    bottom: -1px;
+    line-height: normal;
+    font-weight: bold;
+    font-size: 13px;
+    z-index: 2;
+    opacity: 0;
+    transition: opacity 1s;
+    &:after {
+      content: '';
+      position: absolute;
+      border-width: 5px;
+      border-style: solid;
+      border-radius: 3px;
+      color: $tooltip-bg-color;
+      z-index: -1;
+    }
+    &.tooltip-top {
```

Comment:
the constant here should be same for left and right, bottom and top right?
Could it be in constant ?

Reply:
agree, fixed